lambdas/commands.py

```diff
@@ -136,23 +136,8 @@ def _etl_v2_generate_transform_commands_method(
     return {"files-to-transform": files_to_transform}
 
 
-def generate_load_commands(
-    transform_output_files: list[str], run_type: str, source: str, timdex_bucket: str
-) -> dict:
-    """Generate task run command for loading records into OpenSearch."""
-    # NOTE: FEATURE FLAG: branching logic will be removed after v2 work is complete
-    etl_version = config.get_etl_version()
-    match etl_version:
-        case 1:
-            return _etl_v1_generate_load_commands_method(
-                transform_output_files, run_type, source, timdex_bucket
-            )
-        case 2:
-            return _etl_v2_generate_load_commands_method()
```

Comment:
It became evident that both v1 and v2 sharing the `generate_load_commands()` entrypoint was going to be more confusing as the arguments passed have changed substantially.  It was decided to create a `generate_load_commands_v1()` function which maintains the v1 behavior, and now `generate_load_commands()` (defined below) is _by default_ handling the new v2 behavior.

Reply:
A smart approach!